device-types/Raritan/DSX2-16-1.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Raritan
+model: DSX2-16 Console Server
+slug: dsx2-16-1
+part_number: DSX2-16
+comments: Dominion SX II Console Server
+u_height: 1
+is_full_depth: false
```

Comment:
Please add the `airflow` data point if you have/can find it

Reply:
Was unable to find it, sorry